npc/custom/item_options.txt

```diff
@@ -0,0 +1,234 @@
+//================= Hercules Script =======================================
```

Comment:
Actually not sure about the whole custom script. We talked before to drop support for custom folder entirely. Maybe better to remove it here and offer it in forums?
you also forgot parantheses supposed to be included in all commands, this means also in next(); or close();
exceptions here are break and end